fdbcli/BulkDumpCommand.actor.cpp

```diff
@@ -101,14 +102,32 @@ ACTOR Future<UID> bulkDumpCommandActor(Reference<IClusterConnectionRecord> clust
 		}
 		Key rangeBegin = tokens[2];
 		Key rangeEnd = tokens[3];
-		// Bulk load can only inject data to normal key space, aka "" ~ \xff
+		// Bulk dump can only inject data to normal key space, aka "" ~ \xff
```

Comment:
nit: it made more sense when it was bulk load..... 'load' injects data into the keyspace... 'dump' copies the data out.

Reply:
agreed